src/transport_picoquic.cpp

```diff
@@ -855,18 +815,27 @@ PicoQuicTransport::enqueue(const TransportContextId& context_id,
   }
 
   OutData out_data {
-    .bytes = bytes
+    .bytes = std::move(bytes)
   };
 
-  //std::lock_guard<std::mutex> lock(mutex);
-
   const auto &ctx = active_streams.find(context_id);
 
   if (ctx != active_streams.end()) {
     const auto &stream_cnx = ctx->second.find(stream_id);
 
     if (stream_cnx != ctx->second.end()) {
-      if (!stream_cnx->second.tx_data.push(out_data)) {
+
+        /*
+      if (stream_cnx->second.stream_id) {
+          (void) picoquic_mark_active_stream(stream_cnx->second.cnx,
+                                             stream_cnx->second.stream_id,
+                                             1,
+                                             static_cast<StreamContext *>(&stream_cnx->second));
+      } else {
+        (void) picoquic_mark_datagram_ready(stream_cnx->second.cnx, 1);
+      }*/
```

Comment:
Is this needed in the future? We could grab it from source control if we need it again.